WordPress/Classes/Utility/WPWebViewController.m

```diff
@@ -9,19 +9,53 @@
 #import "WPCookie.h"
 #import "Constants.h"
 #import "WPError.h"
+#import "UIAlertView+Blocks.h"
+#import "WordPress-Swift.h"
 
-@class WPReaderDetailViewController;
+
+
+#pragma mark ====================================================================================
+#pragma mark Constants
+#pragma mark ====================================================================================
+
+#pragma mark ====================================================================================
+#pragma mark Constants
+#pragma mark ====================================================================================
```

Comment:
Double pragma header.

By the way, I really think we should try to standardize on pragmas like:

`#pragma mark - Constants`

Just a thought.